File: __Resources/TextFieldApp/TextFieldApp/ViewController.m
//
//  ViewController.m
//  TextFieldApp
//
//  Created by Phil Wright on 5/27/16.
//  Copyright © 2016 Touchopia, LLC. All rights reserved.
//

#import "ViewController.h"

// Step 1 add the delegate protocol in < >

@interface ViewController () <UITextFieldDelegate>

@property (weak, nonatomic) IBOutlet UITextField *dateTextField;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
}

-(BOOL)textFieldShouldReturn:(UITextField *)textField {
    
    NSLog(@"return key pressed");
    
    // This will remove the keyboard from the app
    [self.dateTextField resignFirstResponder];
    
    return YES;
}



@end

File: 23-Forecaster/WeatherApp/WeatherApp/ViewController.swift
//
//  ViewController.swift
//  WeatherApp
//
//  Created by Phil Wright on 6/23/16.
//  Copyright © 2016 Touchopia, LLC. All rights reserved.
//

import UIKit


// MAKE SURE APP TRANSPORT SECURITY IS OFF!

protocol WeatherAPIDelegate : class {
    func passWeather(weather: Weather)
}

class ViewController: UIViewController, WeatherAPIDelegate {
    
    let apiController = APIController()

    override func viewDidLoad() {
        super.viewDidLoad()
        
        // IMPORTANT
        
        self.apiController.delegate = self
        
        self.apiController.fetchWeather("44.343,-111.2323")
        
 
    }
    
    func passWeather(weather: Weather) {
        
        print("I have the weather")
        
    }


}

